pages/docs/tooling/tooling-config-visualization.md

```diff
@@ -65,27 +66,43 @@ precice-config-visualizer-gui precice-config.xml
 
 The application automatically reloads configuration files on change and shows parsing errors at the bottom. This is especially useful for rapid prototyping.
 
-Desktop integration is still lacking with commonly used tools for installin python packages.
-If you want your launcher to pick up the tool, you can save [its desktop file](https://raw.githubusercontent.com/precice/config-visualizer/master/data/org.precice.config_visualizer.desktop) manually to the directory `~/.local/share/applications/`. The directory may need to be created first.
+Desktop integration is still lacking with commonly used tools for installing python packages.
+If you want your launcher to pick up the tool, you can save [this desktop file](https://gist.githubusercontent.com/fsimonis/a08c3771abf808b0534d658bcb563f90/raw/10717f5b2afbf2d3c05ecb9f5c7eeaae4bbe868a/org.precice.configvisualizer.desktop) (right-click > "Save Link As") manually to the directory `~/.local/share/applications/`. The directory may need to be created first.
```

Comment:
Did not try this yet, too many tabs and programs open over here at the moment.

Reply:
_skill issue_ :wink:

Reply:
_you would know!_

(I had to consult [wikiHow](https://www.wikihow.com/What-Does-Skill-Issue-Mean) on understanding what you wrote me and how to respond)

Reply:
Did you logout and login again? Sometimes a restart is necessary to pick this up too.

Reply:
yes, I restarted.

Reply:
```suggestion
If you want your launcher to pick up the tool, you can save [this desktop file](https://gist.githubusercontent.com/fsimonis/a08c3771abf808b0534d658bcb563f90/raw/e091d78c24b04d0fc903b8de4909528628d22b7b/org.precice.configvisualizer.desktop) (right-click > "Save Link As") manually to the directory `~/.local/share/applications/`. The directory may need to be created first.
```

Reply:
Yes, this worked (assuming it's the same you sent me privately).